sources/web/datalab/static/dark.css

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2015 Google Inc. All rights reserved.
```

Comment:
Note: Both this and the 'light.css' file are derived from the 'datalab.css' file, so they are expected to maintain the same copyright header.